public/js/custom-hj.js

```diff
@@ -6,6 +6,12 @@ jQuery(function($) {
     $('.gp-book-online').on('click', function () {
       hj('tagRecording', ['Book Online']);
     });
+    $('.repeat-prescription-online').on('click', function () {
```

Comment:
to it was on iphone 5 for a reveal, not button. so I guess its not an issue here. I'll approve it.